src/pages/developing/svelte-tutorial/wrapping-up.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Wrapping up
+description:
+  Welcome to Carbon! This tutorial will guide you in creating a Svelte app with
+  the Carbon Design System.
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+<PageDescription>
+
+Thanks for completing our tutorial and helping us improve it along the way. If
+you want to show the world your new skills, you can apply for an IBM Digital
+Badge.
```

Comment:
is there/will there be a badge process for this? if not maybe we'd want to replace the content in this page to something similar to https://carbondesignsystem.com/developing/react-tutorial/wrapping-up

Reply:
@francinelucca There will not - we've recently removed badging from all tutorials.